### ISSUE TITLE
The sign-in link should be an icon

### ISSUE THREAD

**Issue by nikeshnazareth** [two-brothers/overcooked-cms#18]
Or at least a different colour. The default purple is hard to see against
the blue app bar.

**Comment by nikeshnazareth** [comment]
I have styled the _SignIn_ link to match the _SignOut_ button. I did this by hardcoding the styles that get assigned to the button on my desktop. Oddly, the sizes don't seem to match up correctly.

This may cause inconsistencies with other devices/browsers. It also means we will need to change the link styles whenever the button styles are changed.

We also don't get any of the hover styles (but I could add them manually too).

I suspect there is a better way. @bigfriedicecream: any ideas?

Note: we can't just use a button by the link has to bypass the React-Router (we're navigating to the /auth route on the server). If there is a way to do that with a button `onClick` event, that would be ideal. Otherwise, is there a way to embed an <a> tag inside a button such that the button click triggers the link.
